R/Connection.R

```diff
@@ -712,6 +712,56 @@ setMethod(
     invisible(TRUE)
   })
 
+#' List Locations of ODBC Configuration Files
+#'
+#' @description
+#' On MacOS and Linux, odbc uses the unixODBC driver manager to manage
+#' information about driver and data sources. This helper returns the filepaths
+#' where the driver manager will look for that information.
+#'
+#' @details
+#' This function is a wrapper around the command line call `odbcinst -j`.
+#'
+#' Windows now uses the ODBC Data Source Administrator application to manage
+#' drivers and thus does not use `.ini` files; this function will return a
+#' 0-row data frame on Windows.
```

Comment:
I think you could shrink this up a bit and put it all in the description.

Reply:
You got it—[680b502](https://github.com/r-dbi/odbc/pull/680/commits/680b50228c201b284d3e93be45b0b2cdcd09cfbb).